hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/upgrade/SixToSevenUpgradeHandler.java

```diff
@@ -37,6 +40,12 @@ public class SixToSevenUpgradeHandler implements UpgradeHandler {
   public Map<ConfigProperty, String> upgrade(HoodieWriteConfig config, HoodieEngineContext context,
                                              String instantTime,
                                              SupportsUpgradeDowngrade upgradeDowngradeHelper) {
+    HoodieTableConfig tableConfig = upgradeDowngradeHelper.getTable(config, context).getMetaClient().getTableConfig();
```

Comment:
Don't we need this for 7->8 upgrade/downgrade handlers too?

Reply:
Added this for 7 <-> 8 version change. Removed it from 6 <-> 7. Since the change is only available in version 8.